prov/rxm/src/rxm.h

```diff
@@ -474,6 +496,39 @@ static inline void rxm_cntr_incerr(struct util_cntr *cntr)
 		cntr->cntr_fid.ops->adderr(&cntr->cntr_fid, 1);
 }
 
+#if ENABLE_DEBUG
+static inline void rxm_cq_log_comp(uint64_t flags)
+{
+	flags &= (FI_SEND | FI_WRITE | FI_READ | FI_REMOTE_READ |
+		  FI_REMOTE_WRITE);
+
+	switch (flags) {
+	case FI_SEND:
+		FI_DBG(&rxm_prov, FI_LOG_CQ, "Reporting send completion\n");
+		break;
+	case FI_WRITE:
+		FI_DBG(&rxm_prov, FI_LOG_CQ, "Reporting write completion\n");
+		break;
+	case FI_READ:
+		FI_DBG(&rxm_prov, FI_LOG_CQ, "Reporting read completion\n");
+		break;
+	case FI_REMOTE_READ:
+		FI_DBG(&rxm_prov, FI_LOG_CQ, "Reporting remote read completion\n");
+		break;
+	case FI_REMOTE_WRITE:
+		FI_DBG(&rxm_prov, FI_LOG_CQ, "Reporting remote write completion\n");
+		break;
+	default:
+		FI_WARN(&rxm_prov, FI_LOG_CQ, "Unknown completion\n");
+	}
```

Comment:
This is just moving code, but you could use fi_tostr() with FI_TYPE_CQ_EVENT_FLAGS